app/utils/cache-control.test.ts

```diff
@@ -0,0 +1,85 @@
+import { describe, it, expect } from 'vitest'
+
+import { parse, CacheControl } from './cache-control'
+
+describe('parse', () => {
+  it('parses basic cache control directives', () => {
+    const result = parse('max-age=3600, public')
+
+    expect(result.maxAge).toBe(3600)
+    expect(result.public).toBe(true)
+  })
+
+  it('parses s-maxage directive', () => {
+    const result = parse('s-maxage=7200')
+
+    expect(result.sharedMaxAge).toBe(7200)
+  })
+
+  it('parses must-revalidate directive', () => {
```

Comment:
be a little more precise. what is the intended behaviour? parsing doesn't tell the reader anything.